src/runtime/runtime.js

```diff
@@ -518,16 +518,15 @@ export class ConfiguredRuntime {
     /** @private @const {!DialogManager} */
     this.dialogManager_ = new DialogManager(this.doc_);
 
-    /** @private @const {!../components/activities.ActivityPorts} */
-    this.activityPorts_ = new ActivityPorts(this.win_);
-
     /** @private @const {!Callbacks} */
     this.callbacks_ = new Callbacks();
 
     //NOTE: 'this' is passed in as a DepsDef.  Do not pass in 'this' before
```

Comment:
I'm not sure I understand this warning. Does it apply to the new line of code?

Reply:
Does that help?